src/views/Login/Login.tsx

```diff
@@ -1,20 +1,28 @@
 import { Button, Grid, TextField } from "@material-ui/core";
-import React from "react";
+import React, { useState } from "react";
+import { login } from "../../services/api";
 
 export const Login = () => {
-  return (
+  const [email, setEmail] = useState<String>('');
+  const [password, setPassword] = useState<String>('');
+  const [isLoading, setIsLoading] = useState<Boolean>(false)
+  function handleSubmit(e:React.FormEvent<HTMLInputElement>){
+    e.preventDefault()
+    login(email, password)
+  } 
```

Comment:
Why you gotta write functions like this

```suggestion
  const handleSubmit = (e: React.FormEvent<HTMLInputElement>) => {
    e.preventDefault()
    login(email, password)
  } 
```